docs/guide/writing-tests-for-modules.md

```diff
@@ -0,0 +1,163 @@
+# Writing tests for modules
+
+## Requirements
+
+- A running MongoDB server
+
+## Setup
+
+This setup assumes you will use the following packages
+
+- [apostrophe](https://www.npmjs.com/package/apostrophe): *ApostropheCMS is a full-featured, open source CMS built with Node.js that seeks to empower organizations by combining in-context editing and headless architecture in a full-stack JS environment*
+- [eslint](https://www.npmjs.com/package/eslint): *ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code*
+- [eslint-config-apostrophe](https://www.npmjs.com/package/eslint-config-apostrophe): *An ESLint configuration for ApostropheCMS core and officials modules*
+- [mocha](https://www.npmjs.com/package/mocha): *Simple, flexible, fun JavaScript test framework for Node.js & The Browser*
+```sh
+npm install --save-dev apostrophe eslint eslint-config-apostrophe mocha
```

Comment:
I'm wondering here if you should specify that they should already have an apostrophe project built, rather than installing apostrophe here. I'm also not sure if installing apostrophe as a dev dependency makes sense. You would know better than me if this would cause any problems in production.

Reply:
I agree, the original paragraph clarified what was going on much better.